TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/magicalitems/umbrasword.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "本影剑",
+    "icon": "theaurorian:umbrasword",
+    "category": "3magicalitems",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "item": "theaurorian:umbrasword",
+            "text": "$(#008cff)本影剑$()会给予玩家 6 秒的伤害吸收、抗性提升、虚弱以及缓慢效果。当你处于胶着状态且十分需要生命值的恢复时相当有用！$(br2)如果有盾牌或其它物品在你的副手上时你需要潜行以激活它。"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
            "text": "$(#008cff)本影剑$()会给予玩家 6 秒的伤害吸收、抗性提升、虚弱以及缓慢效果。当你处于胶着状态且十分需要生命值的恢复时相当有用！$(br2)如果有盾牌或其它物品在你的副手上时你需要潜行以激活它。"
```
删掉的这句话需要实测。